### PR TITLE
Initial bulk update of README

### LINKED ISSUE #16: Update documentation

### PR DESCRIPTION
**Pull request by grwthomps**
closes #16

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,395 @@
...
+This project uses a PostgreSQL database. Ensure you have Postgres installed and create a new database. Naming convention can be changed, however to get up and running as quickly as possible, we recommend naming the database `condensate_dev`. The path to the database needs to be specified in the environment variables (see the *Setting Environment Variables* section below).
+
+#### Setting Environment Variables
+
+For now, environment variables must be set manually for new sessions using the following commands:
+```
+export FLASK_APP="run.py"
+export APP_SETTINGS="development"
+export DATABASE_URL="postgresql://localhost/condensate_dev"
+export FLASK_ENV=development
```
Comment (rhantak) [inline README.md]:
```suggestion
export FLASK_ENV='development'
```
This should be a string

File: README.md [hunk 2]
```diff
@@ -1 +1,395 @@
...
+#### Create a new user:
+
+Sample Request:
+```
+POST /users
+
+Body:
+
+{
+  "first_name": "Keanu"
```
Comment (rhantak) [inline README.md]:
```suggestion
  "first_name": "Keanu",
```

File: README.md [hunk 3]
```diff
@@ -1 +1,395 @@
...
+  "last_name": "Reeves"
```
Comment (rhantak) [inline README.md]:
```suggestion
  "last_name": "Reeves",
```

File: README.md [hunk 4]
```diff
@@ -1 +1,395 @@
...
+  "email": "kreeves@example.com"
```
Comment (rhantak) [inline README.md]:
```suggestion
  "email": "kreeves@example.com",
```

**Comment by rhantak** [comment]
Made a couple suggestions, just small stuff though. Thanks for putting in the work on this!

**Review by rhantak — CHANGES_REQUESTED** [review]

**Review by grwthomps — COMMENTED** [review]
Good catches

**Pull request merged**